libmscore/score.cpp

```diff
@@ -3584,36 +3584,41 @@ ChordRest* Score::findCR(int tick, int track) const
 
 //---------------------------------------------------------
 //   findCRinStaff
-//    find chord/rest <= tick in staff
-//    prefer shortest duration (so it does not overlap next chordrest segment)
+//    find last chord/rest on staff that ends before tick
 //---------------------------------------------------------
 
 ChordRest* Score::findCRinStaff(int tick, int track) const
```

Comment:
True.  It looks like this function was based on findCR(), which *does* use the actual track..  findCRinStaff() was added in https://github.com/musescore/MuseScore/commit/01ebb2e44320f5a2453912870612ac09a639e8ad and it looks like from the beginning it never actually used the track literally, only as a way of finding a staff index.  I guess this had to do with the decision to make spanners other than slurs always live in voice 0, but I don't see a good reason not to convert from track to staff before calling the function.

@wschweer unless you disagree, I can change this parameter.